[2024-07-18T08:01:01.673+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-18T08:01:01.690+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trigger_dag.wait_for_run_file manual__2024-07-18T08:00:59.817398+00:00 [queued]>
[2024-07-18T08:01:01.694+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trigger_dag.wait_for_run_file manual__2024-07-18T08:00:59.817398+00:00 [queued]>
[2024-07-18T08:01:01.694+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-18T08:01:01.699+0000] {taskinstance.py:2330} INFO - Executing <Task(FileSensor): wait_for_run_file> on 2024-07-18 08:00:59.817398+00:00
[2024-07-18T08:01:01.703+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=131) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-18T08:01:01.704+0000] {standard_task_runner.py:63} INFO - Started process 133 to run task
[2024-07-18T08:01:01.705+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_dag', 'wait_for_run_file', 'manual__2024-07-18T08:00:59.817398+00:00', '--job-id', '1559', '--raw', '--subdir', 'DAGS_FOLDER/trigger_dag.py', '--cfg-path', '/tmp/tmpkd8s_ocf']
[2024-07-18T08:01:01.706+0000] {standard_task_runner.py:91} INFO - Job 1559: Subtask wait_for_run_file
[2024-07-18T08:01:01.759+0000] {task_command.py:426} INFO - Running <TaskInstance: trigger_dag.wait_for_run_file manual__2024-07-18T08:00:59.817398+00:00 [running]> on host 4d13026947d2
[2024-07-18T08:01:01.827+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='trigger_dag' AIRFLOW_CTX_TASK_ID='wait_for_run_file' AIRFLOW_CTX_EXECUTION_DATE='2024-07-18T08:00:59.817398+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-18T08:00:59.817398+00:00'
[2024-07-18T08:01:01.828+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-18T08:01:01.829+0000] {baseoperator.py:400} WARNING - FileSensor.execute cannot be called outside TaskInstance!
[2024-07-18T08:01:01.838+0000] {base.py:84} INFO - Using connection ID 'fs_default' for task execution.
[2024-07-18T08:01:01.838+0000] {filesystem.py:85} INFO - Poking for file /opt/airflow/data/small_text
[2024-07-18T08:01:01.839+0000] {filesystem.py:89} INFO - Found File /opt/airflow/data/small_text last modified: 20240708110326
[2024-07-18T08:01:01.839+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-07-18T08:01:01.840+0000] {filesystem.py:85} INFO - Poking for file /opt/airflow/data/small_text
[2024-07-18T08:01:01.840+0000] {filesystem.py:89} INFO - Found File /opt/airflow/data/small_text last modified: 20240708110326
[2024-07-18T08:01:01.840+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-18T08:01:01.850+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=trigger_dag, task_id=wait_for_run_file, run_id=manual__2024-07-18T08:00:59.817398+00:00, execution_date=20240718T080059, start_date=20240718T080101, end_date=20240718T080101
[2024-07-18T08:01:01.887+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-18T08:01:01.914+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-18T08:01:01.915+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
